metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs runit and provides runit_service resource'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '3.0.5'
+version '3.0.6'
```

Comment:
We have tooling that handles the version bump and changelog generation so this will need to be backed out